Tests/LibWeb/Text/input/geometry.html

```diff
@@ -0,0 +1,31 @@
+<script src="include.js"></script>
+<script>
+    test(() => {
+        let testCounter = 1;
+        function test(test) {
```

Comment:
Could this inner `test()` function get a different name? It's a bit confusing when we're using two functions both called `test()`. :sweat_smile: